packages/ui/src/Select/MultiSelectField.tsx

```diff
@@ -165,6 +165,22 @@ export const MultiSelectField = <V extends string | number = number>(props: Mult
     }
   }, [menuIsOpen])
 
+  /** If the size of screen is changed we close dropdown menu to aviod the dropdown menu bug in react-select library.
+   * For more information visit: https://github.com/JedWatson/react-select/issues/3533
+   */
+  useEffect(() => {
+    const handleResize = () => {
+      if (selectRef.current) {
+        selectRef.current.blur()
```

Comment:
Doesn't `selectRef.current.onMenuClose()` work?

Reply:
> Doesn't `selectRef.current.onMenuClose()` work?

It also works but need ReactSelect reference type.